processor/groupbytraceprocessor/documentation.md

```diff
@@ -0,0 +1,71 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# groupbytrace
+
+## Internal Telemetry
+
+The following telemetry is emitted by this component.
+
+### processor_groupbytrace_conf_num_traces
```

Comment:
I think we have an otelcol_ prefix to this metric today. We need to ensure that we are not introducing a breaking change to users.

Reply:
the prometheus exporter configuration prepends that prefix automatically